README.md

```diff
@@ -492,6 +492,30 @@ Available controllers:
 Commanding the robot: see the commands below.
 
 
+### Example 6: "Modular Robots with actuators not providing states and with additional sensors"
```

Comment:
I'm not sure if it's worth mentioning the `slowdown` here? The example sends a '5' command to the velocity controller but it spins much slower (due to a default slowdown of 50)  which may or may not be confusing to people.

Reply:
Slowdown should just make the movement "smoother", i.e., mock dynamics. Otherwise, robot would jump to the goal immediately.